frontend/src/components/filters/descriptions.js

```diff
@@ -359,3 +359,26 @@ export const PROJECT_FILTERS = {
     mode: "multiple",
   },
 }
+
+export const INDEX_FILTERS = {
+  id: {
+    type: FILTER_TYPE.INPUT,
+    key: "id",
+    label: "ID",
+  },
+  name: {
+    type: FILTER_TYPE.INPUT,
+    key: "name",
+    label: "Name",
+  },
+  index_structure: {
```

Comment:
Ordering by structure and set do not work. Might be the same issue as we've had with the last version with the filter name not matching the key...